From: Liu Bo <bo.li.liu@oracle.com>
Date: Mon, 13 May 2013 11:10:44 +0000
Patch-mainline: 3.11
Git-commit: 5798b92d2beb2f7b5378f1f2081a043409fcfae0
References: FATE#312888
Subject: [PATCH] Btrfs: remove useless copy in quota_ctl

We don't need to copy it back to user side as it remains unchanged.

Signed-off-by: Liu Bo <bo.li.liu@oracle.com>
Signed-off-by: Josef Bacik <jbacik@fusionio.com>
Signed-off-by: David Sterba <dsterba@suse.cz>
---
 fs/btrfs/ioctl.c |    3 ---
 1 file changed, 3 deletions(-)

--- a/fs/btrfs/ioctl.c
+++ b/fs/btrfs/ioctl.c
@@ -3721,9 +3721,6 @@ static long btrfs_ioctl_quota_ctl(struct
 		break;
 	}
 
-	if (copy_to_user(arg, sa, sizeof(*sa)))
-		ret = -EFAULT;
-
 	err = btrfs_commit_transaction(trans, root->fs_info->tree_root);
 	if (err && !ret)
 		ret = err;
